js/libs/core.js

```diff
@@ -502,6 +502,21 @@ Fliplet.FormBuilder = (function() {
 
             break;
 
+          case 'flMatrix':
+            if (this.columnOptions.length <= 0) {
```

Comment:
The length can't be less than 0. 

We also shouldn't allow the following scenarios:

- spaces only
- line breaks only
- line breaks and spaces only

This means the row and column options need to be trimmed and cleaned up before validation and before saving.

```suggestion
            if (this.columnOptions.length === 0) {
```

Reply:
@tonytlwu ,

line breaks and spaces scenarios we are handling already [here](https://github.com/Fliplet/fliplet-widget-form-builder/blob/master/js/libs/core.js#L724).

=== change done.